.github/workflows/pre-release.yml

```
@@ -225,6 +242,8 @@ jobs:
             ghcr.io/keptn/${{ matrix.config.artifact }}:${{ env.VERSION }}
           build-args: |
             version=${{ env.VERSION }}
+            buildTime=${{ env.DATETIME }}
+            gitSha=${{ env.GIT_SHA }}
```

you can directly use the outputs here without the need to first set them in the env above
```suggestion
            buildTime=${{ needs.prepare.outputs.datetime }}
            gitSha=${{ needs.prepare.outputs.git_sha }}
```